infra/endorsement_reminder/values-dev.yaml

```diff
@@ -2,3 +2,4 @@ cron: 0 8 * * *
 image: image-registry.openshift-image-registry.svc:5000/d8a8f9-tools/endorsement-reminder:develop
 env: dev
 applicationURL: https://dev.healthprovideridentityportal.gov.bc.ca
+aspnetcoreEnvironment: Development
```

Comment:
`ASPNETCORE_ENVIRONMENT` doesn't make a distinction between a deployed 'dev' environment and a 'test' environment like how our project is set up; the value is `Staging` for both. `Development` in this context refers to local development on a developer's machine.